src/Components/Rockets/RocketItem.jsx

```diff
@@ -0,0 +1,54 @@
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { reserveRocket } from '../../Redux/Rockets';
+import css from './RocketItem.module.css';
+
+// # needed variable from rockets.rockets
+/*
+  rocket_name
+  active
+  description
+  flickr_images
+*/
```

Comment:
- [ ] I recommend removing the commented code here from this file to make you code clean and dry ✨🔥🚀💯